doc/api/prev_api_changes/api_changes_3.1.0.rst

```diff
@@ -531,7 +531,8 @@ The following miscellaneous API elements have been removed
 
     import logging
     logger = logging.getLogger('matplotlib')
-    logger.set_level(logging.INFO)
+    logger.addHandler(logging.StreamHandler())
+    logger.setLevel(logging.INFO)
```

Comment:
```suggestion
    logger.setLevel(logging.INFO)
    # configure log handling: Either include it into your ``logging`` hierarchy,
    # e.g. by configuring a root looger using ``logging.basicConfig()``,
    # or add a standalone handler to the matplotlib logger:
    logger.addHandler(logging.StreamHandler())
```

Usually, you don't add handlers to specific loggers, but just configure the root logger (e.g. via `logging.basicConfig()`, which will catch all log messages. For debugging Matplotlib it might still be ok, do add a dedicated hander. Logging is very flexible but also a bit complicated in Python. Which sulution suits you best depends on your application.

Reply:
I don't have a huge preference here vs. `basicConfig()` (I have an aversion to `basicConfig()` for libraries because it overrides choices made  by user applications.)--in this case user scripts would be using it. I just want to make sure that whatever we have here results in logging messages displayed somewhere.

Reply:
I suggested the somewhat lengthy comment before `addHandler()` so that people are aware that something has to be done. `addHandler()` is one solution, but depending on your usecase, you might want something else (e.g. `basicConfig()`).

Reply:
I totally missed that comment suggestion. My bad.